spyder/plugins/pylint/plugin.py

```diff
@@ -31,6 +33,32 @@
 from .widgets.pylintgui import PylintWidget
 
 
+def get_colors():
+    """
+    Get text and previous rate color for the pylint plugin based on
+    the color theme and color scheme currently selected.
+    """
+    ui_theme = CONF.get('color_schemes', 'ui_theme')
+    color_scheme = CONF.get('color_schemes', 'selected')

```

Comment:
Please define constants here too (I can't suggest good names for them though).

Reply:
There is no way to get the colors from the stylesheet? So if we change the colors in the dark theme, for example, you need to change those here too. If you couldn't apply directly the style sheet here, try to instantiate a styled widget and get the colors from it. What do you think?